src/Microcosm.js

```diff
@@ -226,7 +234,8 @@ Microcosm.prototype = {
    * @param {function} [callback] - A callback to execute after the application starts
    * @return {Microcosm} The invoking instance of Microcosm
    */
-  start(callback) {
+  start(callback=throwIfError) {
+    this.started = true
     install(this.plugins, callback)
     return this
   }
```

Comment:
Aye, it is. It would be called in the `state` plugin.

This execution order isn't intuitive. I'm going to think more on that. In the mean time, I think this is sufficient.